source/community/report-a-bug.md

```diff
@@ -8,36 +8,21 @@ authors:
 
 # Reporting a Bug in oVirt
 
-For reporting bugs, the oVirt project uses the Bugzilla bugtracker and testing tool.
+For reporting bugs, the oVirt project issues tracker within [GitHub oVirt organization](https://github.com/oVirt).
```

Comment:
Just some English:
Regarding `...project issues tracker...`
Shouldn't it be `...project uses issues tracker...` or `...project uses the issues tracker...`?
Also `'` might be required after the word `issues`.

Reply:
done